testing/scenario_app/run_ios_tests.sh

```diff
@@ -38,8 +38,22 @@ fi
 defaults write com.apple.iphonesimulator RotateWindowWhenSignaledByGuest -int 1
 
 cd $SRC_DIR/out/$FLUTTER_ENGINE/scenario_app/Scenarios
+
+echo "Running simulator tests with Skia"
+echo ""
+
 set -o pipefail && xcodebuild -sdk iphonesimulator \
   -scheme Scenarios \
   -destination 'platform=iOS Simulator,OS=13.0,name=iPhone 8' \
   clean test \
   FLUTTER_ENGINE="$FLUTTER_ENGINE"
+
+echo "Running simulator tests with Impeller"
```

Comment:
Can we skip the golden tests in scenarios app?

Reply:
They have some wiggle room
https://github.com/flutter/engine/blob/1a6005f4f96612607f02b64f9691777eebe83758/testing/scenario_app/ios/Scenarios/ScenariosUITests/GoldenImage.m#L93-L94

The existing ones passed on impeller with no changes, maybe we should keep them and turn it off when we have a good reason?

wdyt @zanderso

Reply:
Let's leave them on. Seeing where/if they fail would help us understand differences between Skia and Impeller.

Reply:
Oh wait, they did fail. I don't know why I thought it was all passing when I wrote that comment.

Because of https://github.com/flutter/flutter/issues/74344#issuecomment-863532692 I can't actually see the diff screenshots though.

Reply:
I'll take a look at this again on Monday.

Reply:
Filed https://github.com/flutter/flutter/issues/113250 and https://github.com/flutter/flutter/issues/113251

Reply:
Thank you!